tap_facebook_pages/client.py

```diff
@@ -143,6 +143,14 @@ def validate_response(self, response: requests.Response) -> None:
                 == "(#190) This method must be called with a Page Access Token"
             ):
                 raise RetriableAPIError(msg)
+            monetization_access_pattern = re.compile(
+                "^.*Monetization metrics are only visible for Page admins.*$"
+            )
+            if response.status_code == 403 and monetization_access_pattern.match(
+                response.json().get("error", {}).get("message")
+            ):
+                self.logger.warning(f"Skipping record because: {msg}")
```

Comment:
Just checking - we don't have access to the video ID here right? Could be useful to log that if we did, but I'm noticing I didn't do that for the post insights stream validation a few lines above

Reply:
Yeah doesn't seem it like, response looks like this - 

<img width="844" alt="Screen Shot 2022-12-28 at 3 25 29 PM" src="https://user-images.githubusercontent.com/14810073/209868222-01444c3b-f203-44a9-b751-13a0e8c98610.png">